source/androidx.activity/activity/Additions/ActivityResultContracts.net80.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+using Android.Runtime;
+using Java.Interop;
+using Java.Lang;
+
+#if ! NET9_0_OR_GREATER
+
+namespace AndroidX.Activity.Result.Contract 
```

Comment:
This file is the same as the `ActivityResultContracts.net100.cs` above, do you need two files checking `NET9_0_OR_GREATER`?

If it just adds a `namespace` can you remove it?

Reply:
This was initial idea to include/exclude files "with generated" code for `net8.0` and `net10.0` in `*.csproj`, based on "extension" `*.net80.cs` for `net8.0` and `*.net100.cs` for `net10.0`. Something like MAUI does for platforms. This was for the case if 

"with generated" code is almost completely cleaned up. I was able to avoid it with just metadata. 

> If it just adds a namespace can you remove it?

I will.